### DISCUSSION TITLE
Minimum Requirements

### DISCUSSION THREAD

**Discussion by mosesman831** [kus/cs2-modded-server#71] [Q&A]
Hi guys,

I want to host this on a free VPS, but it only has 1 OCPU and 1GB of ram, but 50gbs of storage. Will this be enough to run this server?

Thanks

**Comment by kus** [comment]
Hey,

Definitely not.

This is what Valve recommends 

https://developer.valvesoftware.com/wiki/Counter-Strike_2/Dedicated_Servers

This is what I personally use which is a little bit more powerful than you need but I had occasional performance issues with lesser spec

https://gcloud-compute.com/n2-standard-2.html

**Reply by 4fc7c8f789** [reply]
Do you mind sharing what the average monthly cost of your setup is?

**Reply by ALegitCookie** [reply]
> Do you mind sharing what the average monthly cost of your setup is?

lol sorry i replied to this thinking it was someone else please ignore

**Reply by kus** [reply]
About $60USD a month

Although I’m in Australia and we get charged more.

You can get an estimate of pricing here and play with specs 

https://cloud.google.com/products/calculator?dl=CiRhOGUzZjZmZi1iZjE5LTRmODQtOTZhZi1iN2FmNzBhNWQ2NDQQCBokMTI0QzI1QjQtREU3Mi00NUY4LUJDOUMtMzQzQTREMTVDMkNC

**Comment by ALegitCookie** [comment]
> Hi guys,
> 
> I want to host this on a free VPS, but it only has 1 OCPU and 1GB of ram, but 50gbs of storage. Will this be enough to run this server?
> 
> Thanks

you're going to want at LEAST 4 cores and 16GB of ram for a _small_ server.

On the storage side, a regular stock csgo server takes up somewhere between 34-39GB. however, with this CS2-modded-server added on top of that, it's going to be even more than that. 
Also, the more people play on it over time, the server is going to have to store data about those people in local SQLite databases. Things like surf times, ranks, etc... and also any community maps have to be downloaded and stored locally by the server to run them. So expect the size of the installation to slowly grow. I would have at least ~256GB just to be safe. But I guess on a VPS you can just buy more storage in smaller increments as the server grows.
